src/main/resources/i18n/messages.properties

```diff
@@ -411,6 +411,10 @@ monthsAgo={0,number,#} months ago
 yearAgo={0,number,#} year ago
 yearsAgo={0,number,#} years ago
 
+
+settings.languages.restart.title=You changed the language!
```

Comment:
Generally I try to avoid exclamation marks! :)

And I think `Language has been changed.` would be more elegant